pkgs/applications/misc/redshift/default.nix

```diff
@@ -1,13 +1,15 @@
-{ fetchurl, stdenv, libX11, libXrandr, libXxf86vm, libxcb, pkgconfig, python
+{ fetchFromGitHub, stdenv, libX11, libXrandr, libXxf86vm, libxcb, pkgconfig, python
 , randrproto, xcbutil, xf86vidmodeproto, autoconf, automake, gettext, glib
 , GConf, dbus, dbus_glib, makeWrapper, gtk, pygtk, pyxdg, geoclue }:
 
 stdenv.mkDerivation rec {
-  version = "1.9.1";
+  version = "git-20150222";
```

Comment:
I can name it `1.10-git` or something if you want. It's 1.10 with a couple of bugfixes: https://github.com/jonls/redshift/commits/master